examples/plotting/polar.py

```diff
@@ -1,3 +1,12 @@
+'''This example shows hot to use polar coordinates to generate a
+grid of polar plots of rose/rohdonea curves.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.ColumnDataSource, bokeh.models.PolarTransform, bokeh.plotting.figure, bokeh.plotting.figure.line, bokeh.plotting.gridplot
+    :refs: :ref:`ug_topics_contour` > :ref:`ug_topics_contour_polar`
```

Comment:
We stopped doing the "breadcrumbs" style with references, that's just too hard to maintain. Only need to add the "last" reference in the chain:
```suggestion
    :refs: :ref:`ug_topics_contour_polar`
```

Reply:
> We stopped doing the "breadcrumbs" style with references, that's just too hard to maintain. Only need to add the "last" reference in the chain:

Reply:
> Only need to add the "last" reference in the chain

sure!